mmengine/runner/loops.py

```diff
@@ -40,6 +40,7 @@ def __init__(
             max_epochs: int,
             val_begin: int = 1,
             val_interval: int = 1,
+            num_batch_per_epoch: Optional[int] = None,
```

Comment:
Adding a new parameter in the middle position may cause a bc issue. Suggest moving it to the end.